app/controllers/admin/tours_controller.rb

```diff
@@ -13,6 +13,28 @@ def show
     @tour_detail = TourDetail.new
   end
 
+  def create
+    @tour = Tour.new tour_params
+    if @tour.save
+      flash[:success] = t "tour.add.success"
+      flash[:info] = t "tour.tour_detail.be_created"
+    else
+      flash[:danger] = t "tour.add.fail"
```

Comment:
Save lỗi có thể là lỗi validatetion thì nên hiển thị lại màn hình new chứ, sao lại di chuyển đến màn hình show đấy

chỗ này thì @tour.id làm gì có giá trị

Reply:
Dạ em để form new trong cái modal của màn hình show luôn nên em chuyển về màn hình show rồi in ra thông báo để user click lại button nhập lại